packages/mdc-animation/index.ts

```diff
@@ -21,31 +21,18 @@
  * THE SOFTWARE.
  */
 
-enum StandardCssPropertyName {
-  ANIMATION = 'animation',
-  TRANSFORM = 'transform',
-  TRANSITION = 'transition',
-}
-
-enum PrefixedCssPropertyName {
-  WEBKIT_ANIMATION = '-webkit-animation',
-  WEBKIT_TRANSFORM = '-webkit-transform',
-  WEBKIT_TRANSITION = '-webkit-transition',
-}
-
-enum StandardJsEventType {
-  ANIMATION_END = 'animationend',
-  ANIMATION_ITERATION = 'animationiteration',
-  ANIMATION_START = 'animationstart',
-  TRANSITION_END = 'transitionend',
-}
-
-enum PrefixedJsEventType {
-  WEBKIT_ANIMATION_END = 'webkitAnimationEnd',
-  WEBKIT_ANIMATION_ITERATION = 'webkitAnimationIteration',
-  WEBKIT_ANIMATION_START = 'webkitAnimationStart',
-  WEBKIT_TRANSITION_END = 'webkitTransitionEnd',
-}
+type StandardCssPropertyName = (
+    'animation' | 'transform' | 'transition'
+);
+type PrefixedCssPropertyName = (
+    '-webkit-animation' | '-webkit-transform' | '-webkit-transition'
+);
+type StandardJsEventType = (
+    'animationend' | 'animationiteration' | 'animationstart' | 'transitionend'
+);
+type PrefixedJsEventType = (
+    'webkitAnimationEnd' | 'webkitAnimationIteration' | 'webkitAnimationStart' | 'webkitTransitionEnd'
+);
 
 interface CssVendorPropertyMap {
```

Comment:
I know this isn't part of the change, but I think `[key: string]` is loose typing. Since `CssVendorPropertyMap` and `JsVendorPropertyMap` are just used in this file, can you change it to:

```
type CssVendorPropertyMap = {
  [key in StandardCssPropertyName]: CssVendorProperty;
}

type JsVendorPropertyMap = {
  [key in StandardJsEventType]: JsVendorProperty;
}```

Reply:
Done